elements/rh-video/demo/rh-video.html

```diff
@@ -0,0 +1,11 @@
+<rh-video>
+  <img slot="thumbnail" src="https://fakeimg.pl//300x150/282828/eae0d0" alt="Image description"/>
+  <template>
+    <iframe title="Title of video" width="300" height="150" frameborder="0" allowfullscreen src="https://www.youtube.com/embed/9gYLKxJ9NbY"></iframe>
```

Comment:
Can all of the demos use a Red Hat video? Maybe [this one](https://www.youtube.com/watch?v=Hc8emNr2igU) about Red Hat OpenShift AI?

Reply:
Resolved in https://github.com/RedHat-UX/red-hat-design-system/pull/1514/commits/c19452df8d19d7de048c131bcde1dccd1dae2c58.